src/cmd/link/internal/ld/pe.go

```diff
@@ -1426,11 +1444,12 @@ func addPEBaseRelocSym(ctxt *Link, s *sym.Symbol, rt *peBaseRelocTable) {
 }
 
 func addPEBaseReloc(ctxt *Link) {
-	// We only generate base relocation table for ARM (and ... ARM64), x86, and AMD64 are marked as legacy
+	// We only generate base relocation table for ARM (and ... ARM64). x86 and AMD64 are marked as legacy
```

Comment:
nit - this looks like an unintended change.

Reply:
I did intend that. Using a comma after "... ARM64)" was confusing to me. That's meant to be the end of the sentence. Only x86 and AMD64 are marked as legacy systems but the comma makes it misleading. Is this an unnecessary change?